### PR TITLE
deploy prod_branch_* branch to production only

### PR DESCRIPTION
**Pull request by manhhung741**
Change the scheme to deploy:
- Master branch to live.steamforvietnam.net
- Prod_branch_* to live.steamforvietnam.org

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches:
       - master
-      - hdoan/fix-install-design-system
+      - hdoan/*
```
Comment (tuanh118) [inline .github/workflows/deploy.yml]: should we remove this and keep master only?

File: .github/workflows/deploy-production.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+name: build
+
+on:
+  push:
+    branches:
+      # desired syntax: prod_branch_yyyy_mm_dd
```
Comment (tuanh118) [inline .github/workflows/deploy-production.yml:6]: We'll probably need to do several releases per day. So something like `prod_branch_yyyy_mm_dd_count` would be good right?

Comment (manhhung741) [reply .github/workflows/deploy-production.yml:6]: I think just `prod_branch_v*` is good for now

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (manhhung741) [reply .github/workflows/deploy.yml]: yeah I removed it

**Pull request merged**